File: Python Solutions/DiceGame.py
def getInputAndSum():
    a,b,c,d = map(int, input().split())
    sum = a+b+c+d
    return sum

def findWinner(gunnarSum, emmaSum):
    if(gunnarSum > emmaSum):
        print('Gunnar')
    elif(emmaSum > gunnarSum):
        print('Emma')
    else:
        print('Tie')

gunnar = getInputAndSum()
emma = getInputAndSum()
findWinner(gunnar,emma)

File: Kattis-Problems/Java-Solutions/DiceGame.java
import java.util.Scanner;

public class DiceGame {
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		int sumGunnar = diceAndSum(sc);
		int sumEmma = diceAndSum(sc);
		determineWinner(sumGunnar, sumEmma);
	}
	
	public static int diceAndSum(Scanner input) {
		int a = input.nextInt();
		int b = input.nextInt();
		int c = input.nextInt();
		int d = input.nextInt();
		
		int sum = a+b+c+d;
		return sum;
	}
	
	public static void determineWinner(int gunnar, int emma) {
		if(gunnar > emma) {
			System.out.println("Gunnar");
		} else if(emma > gunnar) {
			System.out.println("Emma");
		} else {
			System.out.println("Tie");
		}
	}

}
